File: src/main/java/io/github/wntopia/gikipedia/server/domain/auth/dto/request/DataGsmOAuthCallbackReqDto.java
package io.github.wntopia.gikipedia.server.domain.auth.dto.request;

import org.springframework.web.bind.annotation.BindParam;

public record DataGsmOAuthCallbackReqDto(
    String code,
    String state,
    String error,
    @BindParam("error_description") String errorDescription) {}

File: src/main/kotlin/io/github/wntopia/gikipedia/server/domain/auth/dto/request/DataGsmOAuthCallbackReqDto.kt
package io.github.wntopia.gikipedia.server.domain.auth.dto.request

import org.springframework.web.bind.annotation.BindParam

data class DataGsmOAuthCallbackReqDto(
    val code: String? = null,
    val state: String? = null,
    val error: String? = null,
    @BindParam("error_description") val errorDescription: String? = null,
)
